executable/elf.ksy

```diff
@@ -1791,6 +1792,10 @@ enums:
     32: preinit_array    # Array with addresses of preinit fct
     33: preinit_arraysz  # Size in bytes of DT_PREINIT_ARRAY
     34: symtab_shndx     # Address of SYMTAB_SHNDX section
+    35: relrsz
+    36: relr
+    37: relrent
+    38: encoding
```

Comment:
`DT_ENCODING` is not a real tag value, but a special marker:

https://github.com/kaitai-io/kaitai_struct_formats/blob/06ecf174ef2801750c4480d5986553622e6f512d/executable/elf.ksy#L1787-L1792

Therefore, it must stay commented out as it was (it's not an actual tag, and it may even "collide" with an actual tag on the same numerical value, as seen in the existing state where the `encoding` value is 32). The old `encoding` explanatory comment (shown in the above code snippet) needs to be moved to this new position.

Reply:
Whoops, my bad.